### PR TITLE
Add PululuK contributor interview

### PR DESCRIPTION
**Pull request by LouiseBonnard**
Add a new contributor interview, this time introducing [PululuK](https://github.com/PululuK) to learn about his open source commitment and to thank him for the great job he is doing here!

🎉

If possible, to be posted next Wednesday, on the 10/14.

### CODE REVIEW

File: news/_posts/2020-10-14-contributor-interview-pululu-kinanga-andre.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+**Do you have any advice for first-time PrestaShop contributors?**
+
+You are not alone! PrestaShop has a rather large community and, likely, other users have already encountered the same problems as you. By contributing, you are helping to improve the solution, and this is an opportunity to share directly with the PrestaShop team.
+
+
+**What’s the number one thing you’ve learnt by contributing to Open Source projects?**
+
+I was not alone and there are many ways to look at it!
+
+**Thank you Pululu, we hope to see you more from you as PrestaShop evolves! :)**
```
Comment (atomiix) [inline news/_posts/2020-10-14-contributor-interview-pululu-kinanga-andre.md]:
```suggestion
**Thank you Pululu, we hope to see more from you as PrestaShop evolves! :)**
```

File: news/_posts/2020-10-14-contributor-interview-pululu-kinanga-andre.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+authors: [ LouiseBonnard ]
+icon: icon-leaf
+image: /assets/images/2020/10/Pululu-Kinanga-Andre.png
+tags:
+ - contribute
+ - community
+ - contribution
+---
+
+An open source community is more than just PR made by faceless strangers. In order to better understand the people who contribute time and skills to the PrestaShop project, we're writing a series of interviews with contributors of all ranges. This week, meet [Pululu Kinanga André](https://github.com/PululuK), one of our great developers on GitHub!
```
Comment (PierreRambaud) [inline news/_posts/2020-10-14-contributor-interview-pululu-kinanga-andre.md]:
```suggestion
An open source community is more than just pull requests made by faceless strangers. In order to better understand the people who contribute time and skills to the PrestaShop project, we're writing a series of interviews with contributors of all ranges. This week, meet [Pululu Kinanga André](https://github.com/PululuK), one of our great developers on GitHub!
```

**Review by atomiix — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Review by PululuK — APPROVED** [review]
Thanks !

**Review by NeOMakinG — APPROVED** [review]

**Comment by matthieu-rolland** [comment]
thank you @LouiseBonnard and @PululuK !

**Pull request merged**